scripts/prism.sh

```diff
@@ -1,7 +1,11 @@
 #!/bin/bash
 set -e
-
-LIBRARIES=${LIBRARIES:-go java node csharp}
```

Comment:
As noted above, this can be reverted and simply refer to the `LANGUAGES` env var.